feeds/ca-on.json

```diff
@@ -231,14 +235,17 @@
         },
         {
             "name": "OCTranspo",
-            "type": "transitland-atlas",
-            "transitland-atlas-id": "f-f24-octranspo",
-            "fix": true
+            "type": "http",
+            "url": "https://oct-gtfs-emasagcnfmcgeham.z01.azurefd.net/public-access/GTFSExport.zip",
+            "http-options": {
+                "fetch-interval-days": 1
```

Comment:
The server seems to support the `Last-Modified` header to prevent fetching the file unnecessary often, is the manual limit really necessary?